build_library/sysext_mangle_flatcar-podman

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -euo pipefail
+rootfs="${1}"
+
+pushd "${rootfs}"
+
+rm -rf ./usr/{lib/debug/,lib64/cmake/,lib64/pkgconfig,include/,share/fish,share/aclocal,share/SLSA}
+
+mkdir -p ./usr/share/podman/etc
+cp -a ./etc/{fuse.conf,containers} ./usr/share/podman/etc/
+
+cat <<EOF >>./usr/lib/tmpfiles.d/podman.conf
+C /etc/containers - - - - /usr/share/podman/etc/containers
+C /etc/fuse.conf - - - - /usr/share/podman/etc/fuse.conf
+w /etc/subuid - - - - core:1065536:65536
+w /etc/subgid - - - - core:1065536:65536
+EOF
+
+mkdir -p ./usr/lib/systemd/network
+
+cat <<EOF >>./usr/lib/systemd/network/50-podman.network
+[Match]
+Type=bridge
+Name=podman*
+
+[Link]
+Unmanaged=yes
```

Comment:
Is this really needed or a precaution? The `zz-default.network` should not apply DHCP on bridges.

Reply:
Apparently just a precaution-again me working off of the prior work on this, where this was redundant as well I suppose. Will be removed in the next revision.